apps/dash-financial-report/app.py

```diff
@@ -0,0 +1,80 @@
+# coding: utf-8
+
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+from dash.dependencies import Input, Output
+import plotly.graph_objs as go
+
+from pages import (
+    overview,
+    pricePerformance,
+    portfolioManagement,
+    feesMins,
+    distributions,
+    newsReviews,
+)
+from utils import Header, make_dash_table
+
+import pandas as pd
+
+# import pathlib
+
+app = dash.Dash(
+    __name__, meta_tags=[{"name": "viewport", "content": "width=device-width"}]
+)
+server = app.server
+
+# Describe the layout/ UI, of the app
+app.layout = html.Div(
+    [dcc.Location(id="url", refresh=False), html.Div(id="page-content")]
+)
+
+# Update page
+@app.callback(
+    dash.dependencies.Output("page-content", "children"),
+    [dash.dependencies.Input("url", "pathname")],
+)
+def display_page(pathname):
+    if (
```

Comment:
I think you can remove this `if` statement . You've handled all cases in which a pathname that needs to go to a specific view/page is given, and if one of those pages isn't requested, you want to just redirect to the main app. So, get rid of this `if`, change 

```python
    elif pathname == "/dash-financial-report/price-performance":
```

to 

```python
    if pathname == "/dash-financial-report/price-performance":
```

and leave everything else as it is.